fuel-client/src/client/schema.rs

```diff
@@ -191,6 +191,25 @@ pub struct ContinueTx {
 #[cynic(schema_path = "./assets/schema.sdl")]
 pub struct RunResult {
     pub breakpoint: Option<OutputBreakpoint>,
+    pub json_receipts: Vec<String>,
+}
+
+impl RunResult {
+    pub fn receipts(&self) -> impl Iterator<Item = fuel_tx::Receipt> + '_ {
+        self.json_receipts.iter().map(|r| {
+            serde_json::from_str::<fuel_tx::Receipt>(r).expect("JSON deserialization failed")
```

Comment:
Since we will panic, maybe put something meaningful in the function name such as `unchecked`. Alternatively, or maybe better, we might just return a `Result`

Reply:
It might be better to find a way to perform all the deserialization up-front, rather than lazily when users call these getter functions. Since this type is setup for graphql compatibility, we may need to do the unspeakable and create a duplicate type that is more user friendly and automatically converts the gql response when created.